scripting/get5/recording.sp

```diff
@@ -136,3 +190,30 @@ void SetCurrentMatchRestartDelay(float delay) {
     mp_match_restart_delay.FloatValue = delay;
   }
 }
+
+public int DemoRequestCallback(Handle request, bool failure, bool requestSuccessful,
+                        EHTTPStatusCode statusCode) {
+  if (failure || !requestSuccessful) {
+    LogError("Demo request failed, HTTP status code = %d", statusCode);
+    char response[1024];
```

Comment:
You cannot do this. You don't know the length of the body. Do this:

```c
int responseSize;
SteamWorks_GetHTTPResponseBodySize(request, responseSize);
char[] response = new char[responseSize];
SteamWorks_GetHTTPResponseBodyData(request, response, responseSize);
```

You also have this response code twice, so maybe you can if/else it a bit better.